Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:14
```

Comment:
Why did you choose an old node version? Pick a newer one if possible please 😊

Reply:
I was using Node container 14 for testing as it was the only version I had installed on my local environment. Using this older version was not ideal, and I agree that upgrading to a newer version would be better. This was an oversight on my part, and bumping up the Nodeversion should not break anything. Thanks for pointing this out!